src/components/SignInButtons/ButtonBase.js

```diff
@@ -17,12 +17,12 @@ const defaultProps = {
 };
 
 const ButtonBase = ({onPress, icon}) => (
-    <Pressable
+    <PressableWithoutFeedback
```

Comment:
Notes on this component: we should retest a11y with it implemented, the generic pressable seems to set some aria attributes which should usually only affect web, but RN started implementing web props in recent versions as well and I'm not sure if that applies to aria props